packages/ndla-ui/src/NDLAFilm/SlideshowIndicator.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import BEMHelper from 'react-bem-helper';
+import { NDLAMovie } from './interfaces';
+
+interface Props {
+  slideshow: Array<NDLAMovie>;
+  activeSlide: number;
+  gotoSlide: Function;
```

Comment:
Prøv å unngå Function, men heller definer signaturen
```suggestion
  gotoSlide: (indexTarget: number, useAnimation: boolean) => void;
```